docs/main_text.md

```diff
@@ -558,6 +558,16 @@ Below is the detailed folder structure:
 
 ![docs-downloads-quantpendia](https://user-images.githubusercontent.com/15315514/65271488-4289af00-daeb-11e9-9006-2d7536c4a103.png)
 
+# Childhood Cancer Collection
+
+Childhood Cancer Collections (CCC) is a curated collection of pediatric cancer research-relevant datasets available from refine.bio.
+Childhood Cancer Data Data Lab staff members select the dataset in CCC based on criteria such as the information contained in a submitter-supplied abstract, age ranges associated with samples in an experiment, and tagged ontology terms.
```

Comment:
The content of this sentence is great, but I'm having some trouble with the singular/plural thing in it. I think we want to make "dataset" plural, as I have below, but then I want to make the abstracts plural too, but then we have "samples in an experiment" which I don't quite know what to do with. Maybe just delete "in an experiment"? I think that is what I would go with, and have suggested below. But maybe you have other ideas?


```suggestion
Childhood Cancer Data Data Lab staff members select the datasets in CCC based on criteria such as the information contained in submitter-supplied abstracts, age ranges associated with samples, and tagged ontology terms.
```

Reply:
I think what you have is almost perfect, but we should use "experiments" instead of "datasets" (my error) to be more consistent with how we use the term dataset elsewhere (i.e., to refer to whatever folks put in their cart!)
```suggestion
Childhood Cancer Data Data Lab staff members select the experiments in CCC based on criteria such as the information contained in submitter-supplied abstracts, age ranges associated with samples, and tagged ontology terms.
```